geometry_msgs/PoseStamped pose_stamped
string arm

---

bool succeeded